### ISSUE TITLE
[A11y 8/25] Missing label

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-menu-button#23]
With a screen reader, this is largely okay, but the menu buttons don't have any label. They just say "button". It would be good to add a label here saying "menu button".
